peregrine/analysis/tracking_loop.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env python
```

Comment:
It is a good idea to standardize the set of input arguments and use it both in tracking_loop.py and run.py. Indeed, initSettings.py could be removed altogether in that case. I will try to harmonize them.